apis/instrumentation/v1alpha1/instrumentation_types.go

```diff
@@ -28,6 +28,11 @@ type InstrumentationSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	ResourceAttributes map[string]string `json:"resourceAttributes,omitempty"`
 
+	// Propagators defines inter-process context propagation configuration.
+	// +optional
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	Propagators []string `json:"propagators,omitempty"`
```

Comment:
Isn't there an annotation that generates some validation of the possible values? I know we have that for enums, but I wonder if there's something for a list of values?